### PR TITLE
app-emulation/wine-{vanilla,staging}: revert winegcc '=' parameter handling

### PR DESCRIPTION
**Pull request by trofi**

### CODE REVIEW

**Comment by sarnex** [comment]
Sorry if I should know this. I remember when you first tried to port this to wine_staging, the build failed. Does your recent patch from https://github.com/gentoo/gentoo/pull/21609 get it working?

**Comment by trofi** [comment]
Yes, https://github.com/gentoo/gentoo/pull/21609 was a prerequisite to get `wine-staging` built on my `binutils-config[-native-symlinks]` system.

File: app-emulation/wine-staging/wine-staging-6.12-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,666 @@
...
+usr/share/applications/wine-notepad.desktop
+usr/share/applications/wine-uninstaller.desktop
+usr/share/applications/wine-winecfg.desktop"
+
+PATCHES=(
+	"${PATCHDIR}/patches/${MY_PN}-5.0-winegcc.patch" #260726
+	"${PATCHDIR}/patches/${MY_PN}-4.7-multilib-portage.patch" #395615
+	"${PATCHDIR}/patches/${MY_PN}-2.0-multislot-apploader.patch" #310611
+	"${PATCHDIR}/patches/${MY_PN}-5.9-Revert-makedep-Install-also-generated-typelib-for-in.patch"
+
```
Comment (sarnex) [inline app-emulation/wine-staging/wine-staging-6.12-r1.ebuild:175]: Extra space

File: app-emulation/wine-staging/wine-staging-6.12-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,666 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
```
Comment (sarnex) [inline app-emulation/wine-staging/wine-staging-6.12-r1.ebuild:2]: Do we really need a revbump for this? If the symptom is compile failure, I don't think so

File: app-emulation/wine-staging/wine-staging-6.12-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,666 @@
+# Copyright 1999-2021 Gentoo Authors
```
Comment (sarnex) [inline app-emulation/wine-staging/wine-staging-6.12-r1.ebuild:1]: Comments go for changes to vanilla and staging

**Review by sarnex — CHANGES_REQUESTED** [review]

File: app-emulation/wine-staging/wine-staging-6.12-r1.ebuild (hunk 1 above)
Comment (trofi) [reply app-emulation/wine-staging/wine-staging-6.12-r1.ebuild]: Done.

File: app-emulation/wine-staging/wine-staging-6.12-r1.ebuild (hunk 3 above)
Comment (trofi) [reply app-emulation/wine-staging/wine-staging-6.12-r1.ebuild]: Done.

**Review by sarnex — APPROVED** [review]
Thanks! Let me know when you push so I can deal with ::wine

File: app-emulation/wine-staging/wine-staging-6.12-r1.ebuild (hunk 2 above)
Comment (trofi) [reply app-emulation/wine-staging/wine-staging-6.12-r1.ebuild]: It depends on how much we care about installed `winegcc`'s behaviour. Mechanically we do change installed driver's behaviour. Perhaps nobody uses it with multiple `=` options if we don't have reports yet. I reverted the revbump.

Comment (sarnex) [reply app-emulation/wine-staging/wine-staging-6.12-r1.ebuild]: Thanks for explanation. I don't think many people use winegcc, so I'm fine with no revbump

**Comment by trofi** [comment]
Pushed to `::gentoo`.

**Comment by sarnex** [comment]
Thanks

**Pull request closed without merging**